sklearn/ensemble/_hist_gradient_boosting/binning.py

```diff
@@ -25,15 +25,18 @@ def _find_binning_thresholds(data, max_bins, subsample, random_state):
     ----------
     data : array-like, shape (n_samples, n_features)
         The data to bin.
+
```

Comment:
(@sadakmed For now, this comment is not actionable.)

I do not think we have a consistent policy on if we want a new line between each parameter.

@glemaitre @NicolasHug Have we decided this somewhere before?

Reply:
I don't think that we made any decision. I think this is more common to have line breaks in our docstring but we don't have any in the histogram-gradient boosting and they don't have any in `numpydoc` example.

IMO, both ways would be fine: one would make things longer but slightly more readable while the other will be much more compact but slightly less readable from time to time (I might be leaning towards the latter).